src/logger.ts

```diff
@@ -1,8 +1,56 @@
 /**
- * A method that returns the default logger for this application.
- * Uses a function instead of a constant to allow mocking.
- * @returns The default logger for this application
+ * The default logger for this application.
+ * Can be set to any object that has `debug`, `info`, `warn`, and `error` methods,
+ * as long as it can be cast to a `Console`.
+ * @public
```

Comment:
Doesn't the later `export` imply `@public`?

What do we mean by "public," anyway? This project isn't an API or SDK, the object is only ever used internally

Reply:
It's more for consistency's sake. I like verbose documentation, and some modules have a mixed of internally-used methods and exported methods - and some have methods that are technically private but exported for testing reasons. I just like adding the tag to the code I write.

Reply:
Fair enough, carry on